### ISSUE TITLE
2nd Call to LivePublisher.setVideoParam() causes crash in native library

### ISSUE THREAD

**Issue by nick-fedesna** [NodeMedia/NodeMediaClient-Android#10]
Discovered in v1.2.6, not present in v1.2

STEPS
1. publish
2. stop publish
3. try change video params
4. crash

~~~
*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
Build fingerprint: 'motorola/victara_verizon/victara:5.1/LPES23.32-25-5-2/2:user/release-keys'
Revision: 'p2bf'
ABI: 'arm'
signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x200
    r0 b88c23b8  r1 001312d0  r2 00249f00  r3 00000210
    r4 00000049  r5 756d3508  r6 00000500  r7 000002d0
    r8 12c53e00  r9 b85df7b8  sl 12ed7da0  fp 13264fa0
    ip a2441cad  sp be98df68  lr a2441ccb  pc a2446228  cpsr 200f0030

backtrace:
    #00 pc 0005d228  /lib/arm/libNodeMediaClient.so (RtmpPublisher_setVideoBitrate+23)
    #01 pc 00058cc7  /lib/arm/libNodeMediaClient.so (Java_cn_nodemedia_LivePublisher_setVideoParam+26)
~~~

**Comment by illuspas** [comment]
This problem has been fixed,release in the next version.

**Comment by illuspas** [comment]
Fix in v1.2.8

**Comment by nick-fedesna** [comment]
confirmed, 1.2.8 fixes, thanks
